src/main/java/com/datadoghq/datadog_lambda_java/DDLambda.java

```diff
@@ -1,5 +1,11 @@
 package com.datadoghq.datadog_lambda_java;
 
+import io.opentracing.Scope;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.Tracer.SpanBuilder;
```

Comment:
If a user hasn't installed the agent, do the open tracing libs have any effect on performance?

Reply:
I suspect the difference is sub-millisecond, but I'll verify that.

Reply:
OK, confirmed. The difference between the two isn't significant.